app/views/sections/_feedback_bar.html.erb

```diff
@@ -0,0 +1,20 @@
+<div class="feedback-bar">
+  <section class="feedback-bar__inner limit-content-width">
+    <div class="feedback-bar__left">
+      <h2 class="blue small">FEEDBACK</h2>
+    </div>
+    <div class="feedback-bar__right">
+      <div id="page-helpful" data-controller="page-helpful">
+        <div>
+          <strong data-page-helpful-target="text">Is this page helpful?</strong>
+          <a href="javascript:void(0)" class="page-question__answer strong" data-page-helpful-target="link" data-action="page-helpful#answer">Yes</a>
+          <a href="javascript:void(0)" class="page-question__answer strong" data-page-helpful-target="link" data-action="page-helpful#answer">No</a>
+        </div>
+      </div>
+      <div>
+        We'd like your feedback on our new Get into teaching website.
+        <a target="_blank" rel="noopener noreferrer" href="https://docs.google.com/forms/d/e/1FAIpQLSdBXbU5nwP6_3TH7HY5rB4ehkSDNzfKqB5X2G7wG4K6LY97-g/viewform">Tell us what you think</a>
```

Comment:
```suggestion
        <a target="_blank" rel="noopener noreferrer" href="https://docs.google.com/forms/d/e/1FAIpQLSdBXbU5nwP6_3TH7HY5rB4ehkSDNzfKqB5X2G7wG4K6LY97-g/viewform">Tell us what you think</a>.
```

Reply:
Both are good suggestions - I've push the updates.